### PR TITLE
Added group/message relationships

### PR DESCRIPTION
**Pull request by VinV5**
Added group/message relationships like we discussed.

### CODE REVIEW

File: SECP-service/src/test/java/com/visucius/secp/models/MessageTest.java [hunk 1]
```diff
@@ -57,4 +58,16 @@ public void testUser(){
...
+    public void testGroup() {
+        AssertAnnotations.assertField(Message.class, "group",ManyToOne.class, JoinColumn.class);
+
+        ManyToOne m = ReflectTool.getFieldAnnotation(Message.class, "group", ManyToOne.class);
+        assertEquals("ManyToOne: Fetch is not equal", FetchType.LAZY, m.fetch());
+
+        JoinColumn joinColumn = ReflectTool.getFieldAnnotation(Message.class, "group", JoinColumn.class);
+        assertEquals("JoinColumn group: name is not equal", "group_id", joinColumn.name());
+        assertEquals("JoinColumn group: nullable is true", false, joinColumn.nullable());
+    }
```
Comment (mohameddahrouj) [inline SECP-service/src/test/java/com/visucius/secp/models/MessageTest.java]: Looks like we need more tests for the codecov plugin to approve.

Comment (VinV5) [reply SECP-service/src/test/java/com/visucius/secp/models/MessageTest.java]: Is there a way to test code coverage on my local copy before I commit?

Comment (mohameddahrouj) [reply SECP-service/src/test/java/com/visucius/secp/models/MessageTest.java]: Thats a good question, I know its integrate with Github so I think for it to run the code coverage report you have to commit. I could be mistaken

Comment (tosinoni) [reply SECP-service/src/test/java/com/visucius/secp/models/MessageTest.java]: @VinV5 to view the test coverage locally. Follow the steps below.
1. Execute "mvn clean install" to run the tests and generate the coverage reports.
2. Go to this folder SECP\SECP-service\target\site\jacoco on your computer
3. Execute the index.html file
4. You would see the ui for the test coverage there. 👍

**Comment by mohameddahrouj** [comment]
Everything looks good, do you want to add more test cases or shall I accept the PR?

**Comment by VinV5** [comment]
If everything is good then yeah. Maybe @tosinoni should check it over as well?

File: SECP-service/src/test/java/com/visucius/secp/models/MessageTest.java [hunk 2]
```diff
@@ -2,11 +2,17 @@
 
 import static org.junit.Assert.*;
 
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import org.hibernate.validator.constraints.Email;
 import org.junit.Assert;
 import org.junit.Test;
 
 import javax.persistence.*;
+import java.sql.Timestamp;
```
Comment (tosinoni) [inline SECP-service/src/test/java/com/visucius/secp/models/MessageTest.java]: is there a reason why you included these import statements ? I don't think you are using them in your test. :)

Comment (VinV5) [reply SECP-service/src/test/java/com/visucius/secp/models/MessageTest.java]: Oops, nice catch. I was trying some stuff and forgot to remove them.

**Pull request merged**